openqa-clone-and-monitor-job-from-pr

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env perl
+
+# Note that this script is intended for execution via the GitHub action defined in
+# actions/clone-job/action.yaml.
+
+package openqa_clone_and_monitor_job_from_pr;
```

Comment:
This is not needed, as this is a script. An idea on a separate level, why not make it an openQA cli command instead of a wrapper for it?

Reply:
> An idea on a separate level, why not make it an openQA cli command instead of a wrapper for it?

obviously, out of the scope of this PR i'd say

Reply:
Oh, and the openQA repo is already big enough. So I really like to add code that is rather specific to this additional scripts repo (like we did with many of the other scripts contained here). But that also means I cannot really utilize the modules from the openQA repo. Even if the modules would be installed in the normal Perl path (or we adjusted the search path) this would be problematic because when someone would change things on the openQA-side it might break things here (and openQA is not really developed as an API-stable Perl module).